indico/modules/events/contributions/templates/display/speaker_list.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'events/contributions/display/author_list.html' %}
```

Comment:
wouldn't it be better to have a base template that has empty title/list blocks and overriding them for author and speaker list instead of inheriting from the author list